src/App.tsx

```diff
@@ -6,14 +6,17 @@ import Sidebar from "./components/Sidebar";
 import SpotlightContext from "./context/SpotlightContext";
 import "./MenuItem/contextMenuHandler";
 import useContextEvents from "./MenuItem/contextMenuHandler";
+
+import CanvasView from "./components/CanvasView";
```

Comment:
CanvasView and CanvasHeader are the two components you'll need to add to the left panel in the split view